stix2validator/validator.py

```diff
@@ -2,14 +2,14 @@
 """
 
 import os
-import json
 import fnmatch
 import datetime
 from itertools import chain
 from collections import Iterable
 
 from jsonschema import Draft4Validator, RefResolver
 from jsonschema import exceptions as schema_exceptions
+import simplejson
```

Comment:
I think idiomatic usage is `import simplejson as json`, and leave the call sites unchanged.